packages/devtools-launchpad/.babelrc

```diff
@@ -10,7 +10,7 @@
 
     ["module-resolver", {
       "alias": {
-        "react-dom": "react-dom/dist/react-dom",
+        "react-dom": "react-dom/umd/react-dom.development",
```

Comment:
@jasonLaster is the right person to answer that question, my understanding is that this has to do with how we package react within the browser toolbox.